charts/testkube-api/templates/rolebinding.yaml

```diff
@@ -1,11 +1,13 @@
+{{ if .Values.rbac.createRoleBindings }}
+
 apiVersion: {{ include "global.capabilities.rbac.apiVersion" . }}
-kind: ClusterRoleBinding
+kind: RoleBinding
 metadata:
-  name: executors-crb-{{ .Release.Name }}
+  name: executors-crb-{{ .Release.Name }}-{{ .Release.Namespace }}
```

Comment:
do we need the `-{{ .Release.Namespace }}` suffix now that is is namespace-scoped?